lib/coverage-instrumenter.js

```diff
@@ -1,12 +1,52 @@
 'use strict';
 
 var Filter = require('broccoli-filter');
-var Istanbul = require('istanbul');
+var Instrumenter = require('./babel-istanbul-instrumenter');
+var fs = require('fs');
+var path = require('path');
+
+function getPathForRealFile(relativePath, root) {
```

Comment:
This should share the logic from `index.js`, where we are doing a very similar thing.